components/admin_console/permission_schemes_settings/permissions_tree_playbooks.tsx

```diff
@@ -52,6 +30,37 @@ const PermissionsTreePlaybooks = (props: Props) => {
         props.onToggle(props.role?.name || '', ids);
     };
 
+    const groups = [
+        {
+            id: 'playbook_public',
+            permissions: [
+                Permissions.PLAYBOOK_PUBLIC_MANAGE_PROPERTIES,
+                Permissions.PLAYBOOK_PUBLIC_MANAGE_MEMBERS,
+            ],
+        },
+        {
+            id: 'runs',
+            permissions: [
+                Permissions.RUN_CREATE,
+            ],
+        },
+    ];
+
+    if (props.license?.SkuShortName === LicenseSkus.Enterprise) {
```

Comment:
I didn't try to share code between this and the component above, since it's a subtly different set of permissions being shown.